packages/zorb-fridge-contracts/contracts/ZorbFridge.sol

```diff
@@ -59,14 +69,19 @@ contract ZorbFridge is ERC721 {
             );
     }
 
+    /// @notice Freezes a zorb with safeTransferFrom - this does not work without the safe transfer
+    /// @param from from address
+    /// @param tokenId tokenId received
     function onERC721Received(
+        address from,
```

Comment:
this is the operator

Reply:
https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/token/ERC721/IERC721Receiver.sol#L22-L24

Reply:
tests because in this case msg.sender is owner. But if you had msg.sender approve another address as operator, and then that operator made this safeTransfer, it would mint the frozen token to the operator.

Reply:
Makes sense. Fixed